File: Helper/AutoStation/AutoStationAndroid/app/src/main/java/com/application/autostation/network/models/input/Schedule.java
package com.application.autostation.network.models.input;

import com.application.autostation.ui.models.Model;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Schedule implements Model {
    @SerializedName("id")
    @Expose
    private Integer id = 0;
    @SerializedName("time")
    @Expose
    private String time;
    @SerializedName("distance")
    @Expose
    private Integer distance;
    @SerializedName("dayOfWeek")
    @Expose
    private Integer dayOfWeek;
    @SerializedName("fromId")
    @Expose
    private Integer fromId;
    @SerializedName("from")
    @Expose
    private Point from;
    @SerializedName("to")
    @Expose
    private Point to;
    @SerializedName("toId")
    @Expose
    private Integer toId;
    @SerializedName("price")
    @Expose
    private Double price;

    public Integer getFromId() {
        return fromId;
    }

    public void setFromId(Integer fromId) {
        this.fromId = fromId;
    }

    public Integer getToId() {
        return toId;
    }

    public void setToId(Integer toId) {
        this.toId = toId;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getTime() {
        return time;
    }

    public void setTime(String time) {
        this.time = time;
    }

    public Integer getDistance() {
        return distance;
    }

    public void setDistance(Integer distance) {
        this.distance = distance;
    }

    public Integer getDayOfWeek() {
        return dayOfWeek;
    }

    public void setDayOfWeek(Integer dayOfWeek) {
        this.dayOfWeek = dayOfWeek;
    }

    public Point getFrom() {
        return from;
    }

    public void setFrom(Point from) {
        this.from = from;
    }

    public Point getTo() {
        return to;
    }

    public void setTo(Point to) {
        this.to = to;
    }

    public Double getPrice() {
        return price;
    }

    public void setPrice(Double price) {
        this.price = price;
    }
}

File: Helper/AutoStation/AutoStation/AutoStation/Models/Schedule.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace AutoStation.Models
{
    /// <summary>
    /// Расписание
    /// </summary>
    public class Schedule
    {
        public int ID { get; set; }
        /// <summary>
        /// Время
        /// </summary>
        /// <value></value>
        public string Time { get; set; }
        /// <summary>
        /// Расстояние
        /// </summary>
        /// <value></value>
        public double Distance { get; set; }
        /// <summary>
        /// День недели
        /// </summary>
        /// <value></value>
        public DayOfWeek DayOfWeek { get; set; }
        /// <summary>
        /// От 
        /// </summary>
        /// <value></value>
        public int FromId { get; set; }
        public virtual Point From { get; set; }
        /// <summary>
        /// Куда
        /// </summary>
        /// <value></value>
        public int ToId { get; set; }
        public virtual Point To { get; set; }
        /// <summary>
        /// Стоимость
        /// </summary>
        /// <value></value>
        public double Price { get; set; }
    }
}
